### PR TITLE
Add Laravel 10 support

### PR DESCRIPTION
**Pull request by pacoorozco**
Signed-off-by: pacoorozco <paco@pacoorozco.info>

### CODE REVIEW

**Comment by pacoorozco** [comment]
Is there anything I can do to facilitate this merge?

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "type": "project",
     "require": {
         "php": ">=7.2",
-        "illuminate/database": "^5.6|^6.0|^7.0|^8.0|^9.0"
+        "illuminate/database": "^5.6|^6.0|^7.0|^8.0|^9.0|^10.0"
     },
     "require-dev": {
         "orchestra/testbench": "^3.7|^4.0|^5.0|^6.0|^7.0",
```
Comment (stevebauman) [inline composer.json]:
```suggestion
        "orchestra/testbench": "^3.7|^4.0|^5.0|^6.0|^7.0|^8.0",
```

**Review by stevebauman — APPROVED** [review]

**Comment by stevebauman** [comment]
Apologies for the wait on this @pacoorozco. The initial PR got lost in my inbox 🙏 .

A new release will be created shortly!

**Comment by stevebauman** [comment]
Released! Thanks for your patience 🙏

**Comment by pacoorozco** [comment]
Thank you Sir... it unblocks my project upgrade... 🤪

**Pull request merged**